src/uu/ls/src/ls.rs

```diff
@@ -1755,6 +1755,7 @@ struct PathData {
     // Result<MetaData> got from symlink_metadata() or metadata() based on config
     md: OnceCell<Option<Metadata>>,
     ft: OnceCell<Option<FileType>>,
+    // can be used to avoid reading the metadata. Can be also called d_type
```

Comment:
I don't understand what is meant with "Can be also called d_type".

Reply:
i added a link